helm-chart/templates/deployment-promscale.yaml

```diff
@@ -46,9 +50,9 @@ spec:
           ports:
             - containerPort: 9201
               name: metrics-port
```

Comment:
Only semi-related to this change, but this is something that has bit me with many charts I've wanted to use in clusters I've managed using Istio. Istio requires that port names in deployments follow a specific convention, essentially `<protocol>-<name>`, otherwise it breaks some of Istio's functionality. We should either allow for customizing the name, or at the very least, change these names so that they follow that convention, e.g. `http-metrics`, `grpc-otel`.

Reply:
I agree that this should be changed for better compatibility. We can do this afterwards though.

Reply:
Sure thing!